ui/src/dashboards/components/CellEditorOverlay.tsx

```diff
@@ -598,6 +594,12 @@ class CellEditorOverlay extends Component<Props, State> {
 
   private get source(): SourcesModels.Source {
     const {source, sources} = this.props
+    const {selectedSource} = this.state
+
+    if (selectedSource) {
+      return selectedSource
+    }
```

Comment:
same here.

Reply:
with this one, i think it's required since if there is not a selectedSource, other logic is performed that returns something different.